components/gc-minister/index.json-ld

```diff
@@ -0,0 +1,169 @@
+					}
+				],
+				"fr": [
+					{
+						"@type": "source-code",
+						"description": "Exemple de code :",
+						"code": "<section class=\"gc-minister\">\n\t<h2>Ministre(s)</h2>\n\t<div class=\"row wb-eqht-grd\">\n\t\t<div class=\"col-md-6\">\n\t\t\t<div class=\"row\">\n\t\t\t\t<div class=\"pull-right col-md-7 col-xs-12\">\n\t\t\t\t\t<h3><a href=\"#\">L'honorable [prénom et nom de famille]</a></h3>\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-5 col-xs-12\">\n\t\t\t\t\t<img src=\"./img/minister-200x250.jpg\" alt=\"L'honorable [prénom et nom de famille]\">\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-7 col-xs-12\">\n\t\t\t\t\t<p>[Titre officiel du ministre]</p>\n\t\t\t\t\t<ul>\n\t\t\t\t\t\t<li><a href=\"\">Lettre de mandat</a></li>\n\t\t\t\t\t\t<li><a href=\"\">Dossier d'information</a></li>\n\t\t\t\t\t</ul>\n\t\t\t\t</div>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n</section>"
+					}
+				]
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_minister_1",
+			"name": "Minister - Iteration 1",
+			"date": "2024-03",
+			"detectableBy": ".gc-minister",
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": "<section class=\"gc-minister\">\n\t<h2>Minister(s)</h2>\n\t<div class=\"row wb-eqht-grd\">\n\t\t<div class=\"col-md-6\">\n\t\t\t<div class=\"row\">\n\t\t\t\t<div class=\"pull-right col-md-7 col-xs-12\">\n\t\t\t\t\t<h3><a href=\"#\">The Honourable [Minister name]</a></h3>\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-5 col-xs-12\">\n\t\t\t\t\t<img src=\"./img/minister-200x250.jpg\" alt=\"The Honourable [Minister name]\">\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-7 col-xs-12\">\n\t\t\t\t\t<p>[Official title]</p>\n\t\t\t\t\t<ul>\n\t\t\t\t\t\t<li><a href=\"\">Mandate letter</a></li>\n\t\t\t\t\t\t<li><a href=\"\">Briefing book</a></li>\n\t\t\t\t\t</ul>\n\t\t\t\t</div>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n</section>"
+				}
+			]
+		}
+	],
+	"changesets": [
+		{
+			"@id": "_:cs_minister",
+			"name": "Minister or institutional head",
+			"status": "stable",
+			"baseOnIteration": "_:iteration_minister_1",
+			"detectableBy": ".gc-minister",
+			"layout": "Each minister takes up half the container. The image (if available) is on the left",
+			"semantic": "section ((h3 a) + img + p + (ul > li))",
+			"style": "The image has a thin grey border.",
+			"context": "To be used in the institutional landing page template.",
+			"schema": [
```

Comment:
add the i18n section (static) where:
* Minister
* Minister(s)
* Ministre
* Ministre(s)

Feel free to remove the singular form if institution are not allowed.

Reply:
@jmealing I assumed "Minister(s)" would always be the title regardless of how many ministers are in the component. Can you confirm?

Reply:
@duboisp I received confirmation by @jmealing that the title can contain any of the following, which means there is no static value:
- Minister
- President
- Parliamentary secretary
- Associate minister
- Ombudsman
- Management